src/libraries/Microsoft.PowerFx.Core/Functions/Delegation/DelegationStrategies/DelegationValidationStrategy.cs

```diff
@@ -402,6 +402,12 @@ protected bool IsValidCallNodeInternal(CallNode node, TexlBinding binding, Opera
             var callInfo = binding.GetInfo(node);
             if (callInfo?.Function != null && ((TexlFunction)callInfo.Function).IsRowScopedServerDelegatable(node, binding, metadata))
             {
+                // Following condition is only applicable for data verse offline support.
```

Comment:
>// Following condition is only applicable for data verse offline support. [](http://example.com/codeflow?start=16&length=73)

It doesn't appear that we are actually checking anywhere that the DataSource is dataverse and the app is offline enabled.

Reply:
fixed in the pac PR